frontend/package.json

```
@@ -6,9 +6,12 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
```

We want to make sure all files consistently use Prettier formatting in the future.

```suggestion
    "lint": "next lint && prettier --check .",
```